icons/windicss.svg

```diff
@@ -1,3 +1,4 @@
-<svg width="24" height="24" version="1.1" viewBox="0 0 333 333" xmlns="http://www.w3.org/2000/svg">
- <path d="m51.536 139.45a13.525 13.525 0 0 1 0-27.05h108.2a27.05 27.05 0 0 0 0-54.1c-7.4386 0-14.202 2.975-19.071 7.9796a13.148 13.148 0 0 1-9.6026 4.1623 13.157 13.157 0 0 1-9.6026-4.1623c-5.2745-5.2745-5.2745-13.931 0-19.206 9.8736-9.7381 23.399-15.824 38.276-15.824a54.098 54.098 0 0 1 54.1 54.1 54.1 54.1 0 0 1-54.1 54.1zm202.88 27.05a13.525 13.525 0 0 0 0-27.05c-3.787 0-7.1685 1.4875-9.6026 3.9216a13.466 13.466 0 0 1-9.5353 3.9557c-1.7712 0-3.5258-0.34898-5.1615-1.0284a13.455 13.455 0 0 1-4.3738-2.9273c-5.139-5.2745-5.139-13.795 0-19.07 7.304-7.304 17.448-11.902 28.673-11.902a40.573 40.573 0 0 1 28.691 69.266 40.574 40.574 0 0 1-28.691 11.885h-189.35a13.525 13.525 0 0 1 0-27.05zm-13.525 81.15h-189.35a13.525 13.525 0 0 1 0-27.05h189.35a40.573 40.573 0 0 1 28.691 69.266 40.574 40.574 0 0 1-28.691 11.885c-11.226 0-21.369-4.599-28.673-11.902-5.139-5.2755-5.139-13.796 0-19.071a13.47 13.47 0 0 1 19.071 0c2.434 2.434 5.8155 3.9225 9.6026 3.9225a13.525 13.525 0 0 0 0-27.05z" fill="#0EA5E9" style="fill:#03a9f4;stroke-width:.97478"/>
-</svg>
+<svg width="220" height="220" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M115.85 48.404C115.85 29.984 100.865 15 82.445 15c-19.412 0-31.018 14.555-31.018 28.632h19.088c0-3.96 3.703-9.544 11.93-9.544 7.893 0 14.316 6.423 14.316 14.316 0 7.893-5.761 14.316-20.29 14.316H24.5V81.81h51.971c24.521 0 39.379-14.984 39.379-33.405zm0 0C115.85 29.984 100.865 15 82.445 15c-19.412 0-31.018 14.555-31.018 28.632h19.088c0-3.96 3.703-9.544 11.93-9.544 7.893 0 14.316 6.423 14.316 14.316 0 7.893-5.761 14.316-20.29 14.316H24.5V81.81h51.971c24.521 0 39.379-14.984 39.379-33.405zM158.197 171.596c0 18.42-14.985 33.404-33.405 33.404-19.412 0-31.018-14.555-31.018-28.632h19.088c0 3.96 3.703 9.544 11.93 9.544 7.893 0 14.316-6.423 14.316-14.316 0-7.893-7.258-14.317-20.29-14.317H61.147v-19.088h57.671c24.521 0 39.379 14.984 39.379 33.405zm0 0c0 18.42-14.985 33.404-33.405 33.404-19.412 0-31.018-14.555-31.018-28.632h19.088c0 3.96 3.703 9.544 11.93 9.544 7.893 0 14.316-6.423 14.316-14.316 0-7.893-7.258-14.317-20.29-14.317H61.147v-19.088h57.671c24.521 0 39.379 14.984 39.379 33.405z" fill="#41A6EE" />
+  <path d="M163.2 44.45c-21.054 0-35.906 16.011-35.906 37.065H144.2c0-10.465 7.6-18.065 19-18.065s17.1 7.6 17.1 18.065c0 10.466-7.403 19.089-27.55 19.089H24.5v19.088h131.1c29.835 0 43.7-19.177 43.7-38.177 0-21.054-14.25-37.065-36.1-37.065zM49.523 138.191H24.5v19.249h25.023v-19.249z" fill="#41A6EE" />
```

Comment:
Would you mind picking a color from the Material Color Palette for this icon?

You can use this tool to find a color in this palette which looks quite similar to the color of the icon:
https://pkief.github.io/material-color-converter/

For the color `#41A6EE` the tool has found `#03A9F4` as the color of the Material Design Palette which is quite similar. But I leave it up to you which color you choose from the palette. The point here is that I try to use these colors for all of the icons to make it consistent.

Reply:
i used the tool and picked #42A5F5

offtopic question: how to you make the color swatch in github?